tests/bluetooth/bsim_bt/bsim_test_l2cap/src/main_l2cap_ecred.c

```diff
@@ -373,7 +371,10 @@ static void send_sdu_concurrently(void)
 		for (int k = 0; k < L2CAP_CHANNELS; k++) {
 			channels[k].iteration = i;
 			channels[k].bytes_to_send = (k == 0) ? LONG_MSG : SHORT_MSG; /* assign less bytes to be sent for the channel 1 */
-			k_work_submit_to_queue(&work_queue[k], &channels[k].work);
+			int err = k_work_submit_to_queue(&channels[k].work_queue, &channels[k].work);
+			if (err < 0){
```

Comment:
I think this will fail conformance test unless you do
```suggestion
			int err = k_work_submit_to_queue(&channels[k].work_queue, &channels[k].work);
			
			if (err < 0){
```

or
```suggestion
			err = k_work_submit_to_queue(&channels[k].work_queue, &channels[k].work);
			if (err < 0){
```
and declare `err` top of top.